src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -1089,6 +1089,68 @@ internal void SelectAllCategories(object parameter)
             }
         }
 
+        /// <summary>
```

Comment:
Interesting. Thanks for it. 
But my question here, in CollapseRecursiverly, you have all the categories.
why can't you get categories.where(x => x.IsExpanded = true); This gives the category that is expanded.
Now for that category get the subcategory and say IsExpanded = false, and then set IsExpanded = false on that category.

I am sorry that I still don't understand the need for searching all the categories here.